test/PHPMailerTest.php

```diff
@@ -2675,6 +2675,131 @@ public function testOAuth()
         $this->assertNull($subject);
         $this->assertInstanceOf(OAuth::class, $PHPMailer->getOAuth());
     }
+
+    /**
+     * Test ICal method
+     */
+    public function testICalMethod()
+    {
+        $this->Mail->Subject .= ': ICal method';
+        $this->Mail->Body = '<h3>ICal method test.</h3>';
+        $this->Mail->AltBody = 'ICal method test.';
+        $this->Mail->Ical = 'BEGIN:VCALENDAR'
+            . '\r\nVERSION:2.0'
+            . '\r\nPRODID:-//PHPMailer//PHPMailer Calendar Plugin 1.0//EN'
+            . '\r\nMETHOD:CANCEL'
+            . '\r\nCALSCALE:GREGORIAN'
+            . '\r\nX-MICROSOFT-CALSCALE:GREGORIAN'
+            . '\r\nBEGIN:VEVENT'
+            . '\r\nUID:201909250755-42825@test'
+            . '\r\nDTSTART;20190930T080000Z'
+            . '\r\nSEQUENCE:2'
+            . '\r\nTRANSP:OPAQUE'
+            . '\r\nSTATUS:CONFIRMED'
+            . '\r\nDTEND:20190930T084500Z'
+            . '\r\nLOCATION:[London] London Eye'
+            . '\r\nSUMMARY:Test ICal method'
+            . '\r\nATTENDEE;CN=Attendee, Test;ROLE=OPT-PARTICIPANT;PARTSTAT=NEEDS-ACTION;RSVP='
+            . '\r\n TRUE:MAILTO:attendee-test@example.com'
+            . '\r\nCLASS:PUBLIC'
+            . '\r\nDESCRIPTION:Some plain text'
+            . '\r\nORGANIZER;CN="Example, Test":MAILTO:test@example.com'
+            . '\r\nDTSTAMP:20190925T075546Z'
+            . '\r\nCREATED:20190925T075709Z'
+            . '\r\nLAST-MODIFIED:20190925T075546Z'
+            . '\r\nEND:VEVENT'
+            . '\r\nEND:VCALENDAR';
+        $this->buildBody();
+        $this->Mail->preSend();
+        $this->assertRegExp(
+            '/Content-Type: text\/calendar; method=CANCEL; charset=utf-8/',
```

Comment:
The default charset is `iso-8859-1`, so this won't match. Either change the match, or add `$this->Mail->CharSet = PHPMailer::CHARSET_UTF8;` to the test script.

You'll need to change the other instances of these problems too.